_dev/src/components/merchant-center-account/phone-verification/phone-verification-popin.vue

```diff
@@ -215,6 +244,42 @@ export default {
     getPhoneNumber() {
       this.phoneNumber = this.$store.getters['accounts/GET_SHOP_INFORMATIONS'].store.phone;
     },
+    resetVerificationCodeInputs() {
+      this.inputsVerificationCode = this.inputsVerificationCode
+        .map(() => ({
+          invitationId: null,
+        }));
+    },
+    goToNextInput(key) {
+      if (key.keyCode === 8) {
+        this.disableSendCodeButton = true;
+        this.isCodeValid = null;
+      }
+      const isFillCompletely = [];
```

Comment:
With the name of this variable, I first expected a boolean in here.

You could compare the number of filled input with the total number of inputs.

```suggestion
      const isCompletelyFilled = this.inputsVerificationCode.length === this.inputsVerificationCode.filter((partialCode) => Number.isInteger(partialCode)).length;
```